### DISCUSSION TITLE
Hottop B-2K+ weird behavior after DROP

### DISCUSSION THREAD

**Discussion by RickGroszkiewicz** [artisan-roaster-scope/artisan#1622] [Q&A]
My Hottop started behaving weirdly this year. I don't know if this is related to artisan, or if it is a hardware problem. I have been using artisan version 2.8.4 since December 2023.

About 50% of the time, my fan would go to zero about five seconds after I click on Drop. This started some time in April 2024. When I compare my alarm files, I see no differences for roasts where this does NOT happen:

<img width="694" alt="sample artisan alarm file (post-roast)" src="https://github.com/artisan-roaster-scope/artisan/assets/25457554/b0eafdee-f664-4f5a-915d-cb146cc3b4c8">

I decided to update to artisan version 2.10.2 to "see what happens". I was surprised that all artisan settings for the new version appear to match my prior settings. Just to be sure, I did a roast this morning. 

The fan did not turn off after I clicked DROP. Instead, the roaster turned off completely after about 30 seconds! I want the roaster to continue cooling the beans in the cooling tray until I exit artisan. 

I hope there is some new setting for version 2.10.2 that controls this post-DROP behavior.

**Comment by MAKOMO** [comment]
No there is now new setting which controls the switch off of the machine. This is done by the machine itself once communication drops. Newer Artisans drop any communication on OFF. Just keep Artisan recording while you are cooling down the beans. It does not hurt!

**Reply by RickGroszkiewicz** [reply]
Thanks, Marko. It just makes the graph look ugly when I don't click on OFF.

**Reply by MAKOMO** [reply]
Why is that? In auto axis mode only the part between CHARGE and DROP is shown: https://www.youtube.com/watch?v=QRAzYkmGYVw

**Reply by RickGroszkiewicz** [reply]
Okay - I have not used that option before, so I will change my default profile for artisan.
